src/starkware/starknet/cli/starknet_cli.py

```diff
@@ -106,12 +106,30 @@ async def invoke_or_call(args, command_args, call: bool):
         raise ValueError('Invalid address format.')
     for abi_entry in abi:
         if abi_entry['type'] == 'function' and abi_entry['name'] == args.function:
+            previous_input = None
```

Comment:
```suggestion
            previous_felt_input = None
```
It is enough to keep the last felt input :)

Reply:
Good point :)